src/tools/genpolicy/genpolicy-settings.json

```diff
@@ -347,6 +347,14 @@
                 "^127\\.(?:[0-9]{1,3}\\.){2}[0-9]{1,3}$"
             ]
         },
+        "AddARPNeighborsRequest": {
+            "forbidden_device_names": [
+                "lo"
```

Comment:
@burgerdev, thanks for looking into this!

Do you think forbidden_device_names for both AddARPNeighborsRequest and UpdateRoutesRequest should specify regex values instead of the full interface names?

Users might not need the flexibility of regex, but I guess some of them might want to specify as forbidden:

"^.*$", or
"^eth[0-9]+$", or even 
"^lo0?$"